pkg/task/identity.go

```diff
@@ -45,15 +42,15 @@ type taskIdentifier struct {
 
 func newTaskIdentifier(identity TaskIdentity) (taskIdentifier, error) {
 	if len(identity.Identity) == 0 {
-		return taskIdentifier{}, fmt.Errorf("missing task identity: can not create task identifier instance")
+		return taskIdentifier{}, fmt.Errorf("failed to create task identifier instance: task id is missing")
```

Comment:
Why is in some places error is returned as fmt error object and some places as VerifyError string?

Reply:
VerifyError is a custom error that is thrown during post template operations if required. If this particular error is thrown & there are retry attempts, then this error is logged and post template operation in retried.